NEWS

```diff
@@ -9,6 +9,13 @@ CHANGES WITH 251:
           can still be used by explicitly configuring RouteTable= setting in
           .netdev files.
 
+        * $MONITOR_METADATA will only be passed to OnFailure=/OnSuccess= handlers
+          if exactly one unit was the trigger for their starting job. Therefore,
```

Comment:
Hmm, so I think this needs to be reworded, because in fact one unit is usually the trigger. We check that just one unit lists the handler unit in its OnFailure/OnSuccess.

Reply:
ok, reworded as suggested